src/main/resources/templates/recipes/create.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.thymeleaf.org/">
+<head th:replace="fragments :: head"></head>
+<body class="container">
+
```

Comment:
Please go into each of these templates and add a line calling up the new navbar. It should look like this:

`<div th:replace="fragments :: page-header"></div>`

Reply:
"These templates" meaning your three recipe templates (create, index, delete).